### DISCUSSION TITLE
LTE Modem connected via USB not recognized as wireless client in WLAN routing

### DISCUSSION THREAD

**Discussion by ShaunFell** [RaspAP/raspap-webgui#1679] [Q&A]
My setup is a RPI 4 with a Waveshare 7600G-H M.2 LTE Modem connected to the RPI via USB. Internet is provided via the LTE hat and I have an access point setup to provide local wireless internet to my devices. I would also like to provide wired internet via the onboard ethernet port. 

I tried following the [manual](https://docs.raspap.com/wlanrouting/) to setup a wired connection, but when I go to the Networking/WLAN Routing page of my web ui, the section `Wireless Client interface` has no options, its greyed out:

![image](https://github.com/user-attachments/assets/311e981d-ec1c-4191-8da0-fa2575ee4afd)

[DHCP Logs](https://pastebin.com/GxVXQrXm)

[RaspAP debug log](https://pastebin.com/HHdpiqdf)

The usb0 interface is the LTE modem providing the internet and it shows up as an ethernet connection on the webui dashboard. Maybe the issue is its not being recognized as a wireless interface?

**Comment by billz** [comment]
@ShaunFell network devices is what you're looking for. check the docs
